src/server/proxy_test.go

```diff
@@ -96,10 +97,11 @@ func proxyTest(t *testing.T, httpClient *http.Client, c *client.APIClient, secur
 	// Test GRPC API.
 	t.Run("TestGRPC", func(t *testing.T) {
 		require.NoErrorWithinTRetry(t, 60*time.Second, func() error {
-			if err := c.CreateProjectRepo(pfs.DefaultProjectName, "test"); err != nil {
+			testRepo := tu.UniqueString("TestGRPC")
+			if err := c.CreateProjectRepo(pfs.DefaultProjectName, testRepo); err != nil {
 				return errors.Errorf("create repo: %w", err)
```

Comment:
It would be good if these errors contained the name of the repo, so when the test fails you can look at the pachd logs and correlate them with this test run.  (One more case below.)